index.html

```diff
@@ -250,9 +250,9 @@ <h3>∞</h3>
             <div class="col-lg-8 col-lg-offset-2">
                 <h2>The Venue</h2>
                 <p>
-                    The Min Kao building is one of the newest buildings on campus, with an expansive interior, modern
+                    The <a href="https://www.eecs.utk.edu/overview/facilities/" target="_blank">Min H. Kao Engineering building</a> is one of the newest buildings on campus, with an expansive interior, modern
```

Comment:
This is a good example since the line above does not pertain to the current issue, but was handled in the previous PR.
When merging the previous PR the line above should dissapear, but normally we should start work on each issue from the current master (cleanly such that diff on PR only has code changes related to it's issue).

For ex: If you're on branch 85 and you do git checkout -b 86, then the 86 branch will track 85 with its current changes as well, and we see the diff include that line as well.

Ideally:
Work on 85: 
- (you're on master) => `git checkout -b 85` => make changes => git add then commit then push to origin's 85
Work on 86: 
- (you're currently on 85) => `git checkout master` (hard reset if needed to be clean after you've already pushed the previous branch to origin) => `git pull` (make sure you're up to date with master before starting work on another branch - if still have local changes in `git status` then reset them with `git reset --hard HEAD`) => `git checkout -b 86` => make changes => then git add commit push to 86

Reply:
yeah it makes sense, I will keep that in mind next time I work on it.